lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +240,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        link_tags = article_bs.find_all('a')
+        for tag in link_tags:
+            url = tag.get('href')
+        return url
 
     def find_articles(self) -> None:
         """
         Find articles.
         """
+        for seed_url in self.get_search_urls():
```

Comment:
how do you control that you have collected enough articles and need to stop crawling?

Reply:
I changed it and put 30 - the number of articles on one page

Reply:
no, there should be no limit per page (you collect EVERYTHING from the page until total number of urls is collected)

Reply:
Done